ddhcpd/files/etc/config/ddhcpd

```diff
@@ -6,19 +6,8 @@ config ddhcpd 'settings'
      option block_timeout '300'
      option tentative_timeout '15'
      option spare_leases '2'
-     option block_network '10.116.220.0/22'
      option dhcp_lease_time '300'
 
-config dhcp_option 'server_identify'
-     option code '54'
-     option len '4'
-     option payload '10.116.254.254'
-
-config dhcp_option 'netmask'
-     option code '1'
-     option len '4'
-     option payload '255.255.128.0'
```

Comment:
`payload` is the value that is set by broadcast in site.conf?

Reply:
Yes. All the options I removed are set up by https://github.com/sargon/gluon-sargon/blob/master/ddhcpd/luasrc/lib/gluon/upgrade/500-ddhcpd

I'd rather like ddhcpd not starting up at all than running with a config not appropriate for your community

Reply:
why is the value `broadcast` just deleted then?

https://github.com/sargon/gluon-sargon/blob/e8930700f8d640dd7d3e4ed5258e859af854ee7c/ddhcpd/luasrc/lib/gluon/upgrade/500-ddhcpd#L21

Reply:
I think it is not optional, but obsolete, it is used nowhere i can find

Reply:
Yes-ish. It is used to remove this section from old configs that do still contain a broadcast section.

ddhcpd can actually interprete that section thus it is not obsolete.

Reply:
OK, but where does it interpret `broadcast`?